learning_journal/models.py

```diff
@@ -18,10 +22,25 @@
 Base = declarative_base()
 
 
-class MyModel(Base):
-    __tablename__ = 'models'
+class Entry(Base):
+    __tablename__ = 'entries'
     id = Column(Integer, primary_key=True)
-    name = Column(Text)
-    value = Column(Integer)
+    title = Column(Unicode(length=255), nullable=False, unique=True)
+    body = Column(Unicode)
```

Comment:
I think what you want here is `UnicodeText`, which is explicitly unlimited in length.